### PR TITLE
ci: fix curvbs buiild in ut.sh

### PR DESCRIPTION
**Pull request by ilixiaocui**
Signed-off-by: ilixiaocui <ilixiaocui@163.com>

### What problem does this PR solve?

Issue Number: #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by ilixiaocui** [comment]
cicheck

**Comment by ilixiaocui** [comment]
cicheck

**Comment by ilixiaocui** [comment]
cicheck

**Comment by ilixiaocui** [comment]
cicheck

**Comment by ilixiaocui** [comment]
cicheck

**Comment by ilixiaocui** [comment]
cicheck

**Comment by ilixiaocui** [comment]
cicheck

**Comment by ilixiaocui** [comment]
cicheck

**Comment by ilixiaocui** [comment]
citest

**Comment by ilixiaocui** [comment]
citest

**Comment by ilixiaocui** [comment]
citest

**Comment by ilixiaocui** [comment]
recheck

**Comment by ilixiaocui** [comment]
cicheck

**Comment by ilixiaocui** [comment]
cicheck

**Comment by ilixiaocui** [comment]
cicheck

**Comment by ilixiaocui** [comment]
citest

**Comment by ilixiaocui** [comment]
cicheck

**Review by xu-chaojie — APPROVED** [review]

File: util/build.sh [hunk 1]
```diff
@@ -186,11 +186,13 @@ build_target() {
...
     fi
 
+    alltarget=`get_target`
     if [ $g_ci -eq 1 ]; then
         g_build_opts+=("--collect_code_coverage")
+        alltarget=...
     fi
 
-    for target in `get_target`
+    for target in $alltarget
```
Comment (xu-chaojie) [inline util/build.sh:195]: if $alltarget = ... ,  will it work？

**Review by wu-hanqing — APPROVED** [review]

File: util/build.sh (hunk 1 above)
Comment (ilixiaocui) [reply util/build.sh:195]: yes, I have tested.

**Pull request merged**